app/controllers/hub/clients_controller.rb

```diff
@@ -97,7 +97,7 @@ def update_take_action
     end
 
     def unlock
-      raise CanCan::AccessDenied unless current_user.admin?
+      raise CanCan::AccessDenied unless current_user.admin? || current_user.org_lead? || current_user.site_coordinator?
```

Comment:
I could be convinced to break this into two lines or a way to make it read easier.